PayloadPkg/Library/MeFwUpdateNullLib/MeFwUpdateNullLib.c

```diff
@@ -0,0 +1,285 @@
+/** @file
+
+  Copyright (c) 2017 - 2019, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+#include <Base.h>
+#include <PiPei.h>
+#include <CsmeUpdateDriver.h>
+
+CSME_UPDATE_DRIVER_INPUT g_FwuCbCtxt;
+
+/**
+  Starting a Full FW Update from a buffer.
+
+  @param[in]  Buffer        Buffer of Update Image read from Update Image file.
+  @param[in]  BufferLength  Length of the buffer in bytes.
+  @param[in]  OemId         OEM ID to compare with OEM ID in FW (if exist). Can be NULL.
+  @param[in]  Func          A callback function that reports the progress of sending the buffer
+                            to FW (not the progress of the update itself). Can be NULL.
+                            void* Func(UINT32 BytesSentToFw, UINT32 TotalBytesToSendToFw);
+  
+  @retval SUCCESS  If Update started successfully. Error code otherwise.
+**/
+UINT32
+FwuFullUpdateFromBuffer(
+  IN  UINT8 *Buffer,
+  IN  UINT32 BufferLength,
+  IN  EFI_GUID *OemId,
+  IN  void(*Func)(UINT32, UINT32)
+  )
+{
+  return 0;
```

Comment:
For dummy API, I think it should return UNSUPPORTED.

Reply:
Removing this whole file in new patch